DeveloperGuide.html

```diff
@@ -0,0 +1,567 @@
+<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]--><meta name="viewport" content="width=device-width, initial-scale=1.0"><meta name="generator" content="Asciidoctor 1.5.6.1"><title>FoodieBot - Developer Guide</title><link rel="stylesheet" href="stylesheets/gh-pages.css">
```

Comment:
Activity diagram under List Implementation is wrong. The first path should join before node "Clear Displayed Content" and before "Add Canteen List for Display"